CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ endif()
 
 ADD_DEFINITIONS(
   -std=c++1z # Or -std=c++0x
```

Comment:
We need to be careful about this, in case we ever want to allow compilation with GCC

Reply:
There's no `--std` flag for gcc compilation, so this might fail.